clients/extension/scripts/popup.js

```diff
@@ -193,7 +201,7 @@ async function onGenerateReportButtonClick(background, settings) {
 
     if (includeReportLink) {
       feedbackEl.className = 'feedback feedback-error';
-      feedbackEl.appendChild(buildReportErrorLink(err));
+      feedbackEl.appendChild(buildErrorCopyButton(/** @type {LighthouseError} */ (err)));
```

Comment:
Ideally we wouldn't do this cast because we don't actually know this...e.g. even if we someday catch all exceptions inside Lighthouse and only throw LighthouseErrors, if an extension API throws, it'll still be a plain old `Error`.

I just meant something like `function buildErrorCopyButton(friendlyMessage, err) {` so we don't have to do the `friendlyMessage || message` dance again inside `buildErrorCopyButton` when we've already figured it out here.